pyshelf/search_parser.py

```diff
@@ -10,10 +10,10 @@ def from_request(self, request_criteria):
             Turns the given request into search criteria that can be consumed by pyshelf.search module.
 
             Args:
-                request_criteria(dict): Search and sort criteria from the request.
+                request_criteria(search-request-criteria.json): Criteria from request.
 
             Returns:
-                dict: search and sort criteria that will be consumed by search layer
+                schemas.search-layer-criteria.json: search and sort criteria that will be consumed by search layer
```

Comment:
It looks like this file is just called search-criteria.json?   Also I have been doing "/" insetad of "." because it is not a python module.  What ya think?